interp/impargs.ml

```diff
@@ -649,7 +649,9 @@ type manual_explicitation = Constrexpr.explicitation * (bool * bool * bool)
 
 type manual_implicits = manual_explicitation list
 
-let compute_implicits_with_manual env typ enriching l =
+let compute_implicits_with_manual env _sigma typ enriching l =
+  (* XXX: FIXME *)
+  let typ = EConstr.Unsafe.to_constr typ in
```

Comment:
Why not to move `sigma` towards `compute_implicits_gen`?

Reply:
Sounds like a good idea @herbelin, what should be done with the whd_all there?

Reply:
Ok, done, I used reductionops.whd_all, but I added a comment to the commit.

Reply:
CC @maximedenes I believe this PR introduced the performance issue Damien showed us yesterday. I'll try to undo this change and use kernel's `whd_all` (that is by need) instead of `Reductionops.whd_all` that is *by name*, and see how it goes.

CC @ppedrot @ejgallego note that the `sigma` threaded there is an empty one, created with `from_env`... https://github.com/coq/coq/pull/6511#issuecomment-354186902